apps/server/net/nanows_client.go

```diff
@@ -80,6 +85,29 @@ func StartNanoWSClient(wsUrl string, callbackChan *chan *WSCallbackMsg) {
 		cancel()
 	}()
 
+	// Goroutine to handle new account addresses
+	go func() {
+		for newAccount := range newAccountChan {
+			//todo: this should use action: "update", "accounts_add" from v21 but node rpc proxies don't support :(
+			addresses = append(addresses, newAccount)
+			// Send update message to WebSocket
+			updateRequest := wsSubscribe{
+				Action: "subscribe",
+				Topic:  "confirmation",
+				Ack:    false,
+				Id:     guuid.New().String(),
```

Comment:
My understanding is the point of the "id" here is for the node to respond with an acknowledgement that it received the message. It would be nice to actually respond to it, or just remove this all together and maybe add a todo type of thing.